lib/gstreamer/va/decoder.py

```diff
@@ -36,6 +36,8 @@ def decode_test_class(codec, bitdepth, **kwargs):
   gstcodec = {
     Codec.AVC   : "h264",
     Codec.HEVC  : "h265",
+    Codec.VVC_8   : "h266",
+    Codec.VVC_10   : "h266",
```

Comment:
```suggestion
    Codec.VVC   : "h266",
```

Reply:
Also, we need to add `Codec.VVC` to the `capcodec` condition (i.e. "caps lookup translation")... located at the top of this function.